saleor/graphql/product/mutations/product_variant/product_variant_update.py

```diff
@@ -167,10 +174,10 @@ def clean_attributes(cls, cleaned_input: dict, instance: models.ProductVariant):
                     attribute_modified = has_input_modified_attribute_values(
```

Comment:
`has_input_modified_attribute_values` will be fixed separately. I hardcoded `attribute_modified` to `True` for now